docs/getting-started-guides/juju.md

```diff
@@ -226,21 +249,28 @@ Or multiple units at one time:
 juju add-unit -n3 kubernetes
 ```
 
+You can also scale the etcd charm for more fault tolerant key/value storage:  
+
+```shell
+juju add-unit -n2 etcd
+```
+
 ## Tear down cluster
 
-We recommend that you use the `kube-down.sh` command when you are done using
+We recommend that you use the `kube-down.sh` script when you are done using
 the cluster, as it properly brings down the cloud and removes some of the
 build directories.
 
 ```shell
-./kube-down.sh
+./cluster/kube-down.sh
 ```
 
-Alternately if you want stop the servers you can destroy your current Juju
-environment. Use the `juju env` command to get the current environment name:  
+Alternately if you want stop the servers you can destroy the Juju model or the
+controller. Use the `juju switch` command to get the current controller name:  
 
 ```shell
-juju kill-controller `juju env`
+juju switch
+juju destroy-controller $controllername
```

Comment:
you'll need to add --destroy-all-models here to ensure we've cleaned up the controller properly. Otherwise, Juju complains and it's not obvious to the user what's going on at this point. We assume zero knowledge of Juju coming into this tutorial.